src/user/dto/create-user.dto.ts

```diff
@@ -1,6 +1,29 @@
-export interface CreateUserDto {
+import { ApiProperty } from '@nestjs/swagger';
+import {
+  IsDateString,
+  IsEmail,
+  IsOptional,
+  IsString,
+  MinLength,
+} from 'class-validator';
+
+export class CreateUserDto {
+  @ApiProperty({ example: 'user@example.com' })
+  @IsEmail()
   email: string;
+
+  @ApiProperty({ example: 'qwerty123' })
+  @IsString()
+  @MinLength(5)
   password: string;
-  name: string;
-  birthday: string;
+
+  @ApiProperty({ example: 'Ivan' })
```

Comment:
```suggestion
  @ApiPropertyOptional({ example: 'Ivan' })
```
Если поле делаешь опциональным, то и в сваггере это нужно указать